### DISCUSSION TITLE
Private Marketplace for VS Code

### DISCUSSION THREAD

**Discussion by seaniyer** [microsoft/vsmarketplace#1069] [Announcement]
Hi all,

If you and your team have the need to host your own private instance of a registry for VS Code extensions,
> ### Please take this [Private Marketplace Survey](https://www.surveymonkey.com/r/2PZXQ5R?src=21839) 

VS Marketplace team is conducting a quick [**survey**](https://www.surveymonkey.com/r/2PZXQ5R?src=21839) to gather the latest customer input on Private Marketplace asks. We are scoping and planning a solution, so your response will be timely in helping shape any Private Marketplace offering from Microsoft. 

Thank you on behalf of VS Marketplace team!

**Comment by CalvinAllen** [comment]
Why is this limited to VSCode only?
